### PR TITLE
Be cute with Option and getenv.

### PR DESCRIPTION
**Pull request by stephenh**
Hey, I saw the getenv commit go by and made some knee-jerk refactorings to use Option.

Feel free to close/ignore this if you think it's more cryptic/not worth it. If so, I'll also avoid small/nitty little PRs like this in the future.

### CODE REVIEW

**Comment by mateiz** [comment]
Jenkins, this is ok to test

**Comment by mateiz** [comment]
Thanks Stephen, this is definitely a lot cleaner!

File: core/src/main/scala/spark/deploy/worker/ExecutorRunner.scala [hunk 1]
```diff
@@ -93,18 +90,8 @@ private[spark] class ExecutorRunner(
...
-      List("-Djava.library.path=" + System.getenv("SPARK_LIBRARY_PATH"))
-    }
-
-    val userOpts = if (System.getenv("SPARK_JAVA_OPTS") == null) {
-      Nil
-    } else {
-      Utils.splitCommandString(System.getenv("SPARK_JAVA_OPTS"))
-    }
-
+    val libraryOpts = Option(getenv("SPARK_LIBRARY_PATH")).map(p => List("-Djava.library.path=" + p)).getOrElse(Nil)
```
Comment (mateiz) [inline core/src/main/scala/spark/deploy/worker/ExecutorRunner.scala]: One problem: this line is longer than 100 chars; just move the .getOrElse below

**Comment by stephenh** [comment]
Ah, right, sorry--was thinking/hoping it was 120. Fixed.

**Comment by mateiz** [comment]
Awesome, thanks!

**Pull request merged**